src/Homework №1/№12_classes.py

```diff
@@ -0,0 +1,20 @@
+# task №12: "Classes"
+
+
+class BankAccount(object):
```

Comment:
you have 0 blank lines between docstring and code in previous tasks and you have 2 here. Your styling should be consistent. 
I suggest 2 lines everywhere

Reply:
not addressed (i.e. task 11)